src/main/java/com/exadel/movie_new/dao/MovieDao.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.movie_new.dao;
+
+import com.exadel.movie_new.model.Movie;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+public interface MovieDao {
+
+    List<Movie> readJsonData();

```

Comment:
read and write - internal (private) methods of the DAO. They are called for each CRUD operation. They should not be in interface